torch/ao/quantization/quantizer/xnnpack_quantizer.py

```diff
@@ -263,60 +279,95 @@ def set_global(self, quantization_config: QuantizationConfig) -> XNNPACKQuantize
         self.global_config = quantization_config
         return self
 
-    def set_config_for_operator_type(
+    def set_operator_type(
         self, operator_type: str, quantization_config: QuantizationConfig
     ) -> XNNPACKQuantizer:
         self.operator_type_config[operator_type] = quantization_config
         return self
 
+    def set_module_name(
+        self, module_name: str, quantization_config: Optional[QuantizationConfig]
+    ):
+        assert (
+            quantization_config is not None
+        ), " quantization_config == None is not supported yet"
+        self.module_name_config[module_name] = quantization_config
+        return self
+
     def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
         """just handling global spec for now"""
         # hacked for handling dynamic linear quant. will fix later.
-        if self.global_config.input_activation.is_dynamic:  # type: ignore[union-attr]
+        if self.global_config and self.global_config.input_activation.is_dynamic:  # type: ignore[union-attr]
             model = self._annotate_for_dynamic_quantization_config(model)
         else:
             model = self._annotate_for_static_quantization_config(model)
         return model
 
+    def _annotate_all_patterns(
+        self,
+        model: torch.fx.GraphModule,
+        config: Optional[QuantizationConfig],
+        filter_fn: Optional[Callable[[Node], bool]] = None,
+    ) -> torch.fx.GraphModule:
+        # TODO: implement the support for None to be canceling out previous annotations
+        if config is None:
+            return model
+
+        assert config is not None
```

Comment:
I feel we don't need this assert given the previous lines :)

Reply:
this is for lint I think